docs/topics/jobs.md

```diff
@@ -0,0 +1,45 @@
+# Jobs
+
+Looking for a Django REST Framework related job? We highlight job vacancies by our awesome sponsors here and also provide a list of other job resources that may be helpful.
+
+
+## Sponsors Who Are Hiring
```

Comment:
Opted to link to the funding site rather than including the logos because I figured it's a better strategy to lead people to that site and possibly get new sponsorship sign ups this way.